forc-plugins/forc-debug/src/server/handlers/handle_next.rs

```diff
@@ -0,0 +1,30 @@
+use crate::server::AdapterError;
+use crate::server::DapServer;
+use forc_test::execute::DebugResult;
+
+impl DapServer {
+    /// Handles a `next` request. Returns true if the server should continue running.
+    pub(crate) fn handle_next(&mut self) -> Result<bool, AdapterError> {
+        self.state.update_vm_breakpoints();
+        if let Some(executor) = self.state.executors.first_mut() {
```

Comment:
This also looks similar to the above code

Reply:
Refactored ✅